install_scripts/container_setup_gw.sh

```diff
@@ -0,0 +1,607 @@
+  skip container_run /usr/bin/systemctl enable --now nic-configuration@host0
+
+  echo '[-] Enable conntrack'
+  container_run /usr/bin/systemctl enable --now conntrack
+
+  echo '[-] Add static ip for container setup'
+  cat >> $CONTAINER_ROOT/etc/systemd/network/10-gw.network <<EOF
+# Extra static ip and route to communicate with the outside world from within the container
+Gateway=10.0.0.1
+Address=10.0.0.254/24
+EOF
+
+  echo '[-] Restart systemd-networkd'
+  container_run /usr/bin/systemctl restart systemd-networkd
+
+  container_snapshot $FUNCNAME
+}
+
+function test_network {
+  echo '[>] Test network... '
+
+  test_service_is_enabled_active conntrack
+
+  echo -n '[>] Check internet access '
+  test_url https://gstatic.com/generate_204
+
+  echo -n '[>] Check gw.prolo IPs '
+  if ! machinectl status $CONTAINER_NAME | grep -q '192.168.1.254'; then
+    echo_ko "FAIL"
+    return 1
+  else
+    echo_ok "PASS"
+  fi
+}
+
+function stage_setup_gw {
+  echo_status 'Run gw setup script'
+
+  container_run /root/sadm/install_scripts/setup_gw.sh
+
+  container_snapshot $FUNCNAME
+}
+
+function stage_setup_mdb {
+  echo_status 'Install mdb'
+  echo '127.0.0.1 mdb' >> $CONTAINER_ROOT/etc/hosts
+  container_run /var/prologin/venv/bin/python install.py mdb
+
+  echo '[-] Enable mdb'
+  container_run /usr/bin/systemctl enable --now mdb
+
+  echo '[-] Reload nginx'
+  container_run /usr/bin/systemctl reload nginx
+
+  # Wait for mdb to start
+  sleep 3
+
+  container_snapshot $FUNCNAME
+}
+
+function test_mdb {
+  echo '[>] Test mdb... '
+
+  test_service_is_enabled_active mdb
+
+  echo -n '[>] GET http://mdb/query '
+  if ! container_run_quiet >&- /usr/bin/curl --fail http://mdb/query; then
+    echo_ko "FAIL"
+    return 1
+  else
+    echo_ok "PASS"
+  fi
+}
+
+function stage_setup_mdbsync {
+  echo_status 'Install mdbsync'
+  container_run /var/prologin/venv/bin/python install.py mdbsync
+  echo '127.0.0.1 mdbsync' >> $CONTAINER_ROOT/etc/hosts
+
+  echo '[-] Enable and start the mdbsync service'
+  container_run /usr/bin/systemctl enable --now mdbsync
+
+  echo '[-] Reload nginx'
+  container_run /usr/bin/systemctl reload nginx
+
+  container_snapshot $FUNCNAME
+}
+
+function test_mdbsync {
+  echo '[>] Test mdbsync... '
+
+  test_service_is_enabled_active mdbsync
+}
+
+function stage_setup_mdbdns {
+  echo_status 'Install mdbdns'
+  container_run /var/prologin/venv/bin/python install.py mdbdns
+
+  container_run /usr/bin/mv /etc/named.conf{.new,}
+  echo '[-] Enable and start the mdbdns service'
+  container_run /usr/bin/systemctl enable --now mdbdns
+
+  container_run /var/prologin/venv/bin/python /var/prologin/mdb/manage.py \
+    addmachine --hostname gw --mac 11:22:33:44:55:66 \
+      --ip 192.168.1.254 --rfs 0 --hfs 0 --mtype service --room pasteur \
+      --aliases mdb,mdbsync,ns,netboot,udb,udbsync,presencesync,ntp
+
+  # Delay for the generated files to be written
+  sleep .5
```

Comment:
Can't we have an active loop to check these file actually exists? This kind of sleeps smells bad.